dotnet.el

```diff
@@ -56,7 +56,7 @@
   "Build a .NET project."
   (interactive)
   (let* ((target (dotnet-select-project-or-solution))
-         (command "dotnet build -v n \"%s\""))
+         (command "dotnet build -v q /p:GenerateFullPaths=true \"%s\""))
```

Comment:
NOTE: this is necessary because the path are relative to the current project root, and not the solution file.